### PR TITLE
Integrate new controls with survey results

### PR DESCRIPTION
**Pull request by mehalshah**
Integrate the new controls with the survey results

- [x] Andrew's changes to use the answer texts instead of answer indicies
- [x] For scale questions, include all responses in range as options
- [x] In results.jsx use the new controls

### CODE REVIEW

**Comment by mehalshah** [comment]
Not included in this change is the work to include other answers - that needs to be done separately

File: apps/src/code-studio/pd/workshop_dashboard/reports/local_summer_workshop_daily_survey/results.jsx [hunk 1]
```diff
@@ -24,167 +16,68 @@ export default class Results extends React.Component {
...
+      } else if (question['answer_type'] === 'text') {
+        return (
+          <TextResponses
+            question={question['text']}
+            answers={answers[questionId]}
+            key={i}
+          />
+        );
+      }
+    }));
```
Comment (aoby) [inline apps/src/code-studio/pd/workshop_dashboard/reports/local_summer_workshop_daily_survey/results.jsx:49]: What about [multi_select](https://github.com/code-dot-org/code-dot-org/pull/23371/files#diff-75f8f172f849ce785da2f56f73ec162fL34)?

File: apps/src/code-studio/pd/workshop_dashboard/reports/local_summer_workshop_daily_survey/results.jsx [hunk 2]
```diff
@@ -24,167 +16,68 @@ export default class Results extends React.Component {
...
-            hasResponses && responses[facilitatorId].map((response, i) => (
-              <li key={i} style={styles.facilitatorResponseList}>
-                {he.decode(response)}
-              </li>
-            ))
-          }
-        </ul>
-      </li>
-    );
+      if (question['answer_type'] === 'scale'|| question['answer_type'] === 'singleSelect') {
```
Comment (aoby) [inline apps/src/code-studio/pd/workshop_dashboard/reports/local_summer_workshop_daily_survey/results.jsx]: suggestion: 
```jsx
if (['scale', 'singleSelect'].includes(question['answer_type']))
```

File: apps/src/code-studio/pd/workshop_dashboard/reports/local_summer_workshop_daily_survey/results.jsx (hunk 1 above)
Comment (mehalshah) [reply apps/src/code-studio/pd/workshop_dashboard/reports/local_summer_workshop_daily_survey/results.jsx:49]: I left this out. We don't have any multi-select questions for the time being.

**Comment by aoby** [comment]
Note that this includes changes from https://github.com/code-dot-org/code-dot-org/pull/23371

**Pull request merged**